src/python/bot/tasks/variant_task.py

```diff
@@ -39,7 +40,9 @@ def execute_task(testcase_id, job_type):
   testcase = data_handler.get_testcase_by_id(testcase_id)
 
   # Setup testcase and its dependencies.
-  file_list, _, testcase_file_path = setup.setup_testcase(testcase)
+  fuzzer_override = builtin_fuzzers.get_fuzzer_for_job(job_type)
```

Comment:
I'm confused why this is called `fuzzer_override`. Shouldn't it just be `fuzzer`

Reply:
Just tries to match fuzzer_override arg on next line
  fuzzer_override = builtin_fuzzers.get_fuzzer_for_job(job_type)
  file_list, _, testcase_file_path = setup.setup_testcase(
      testcase, fuzzer_override=fuzzer_override)

also fuzzer_override makes it clear we are trying to override fuzzer_name based on job_type